nexus/src/db/datastore/organization.rs

```diff
@@ -71,7 +72,16 @@ impl DataStore {
                     ErrorHandler::Conflict(ResourceType::Organization, &name),
                 )
             }
-        })
+        })?;
+
+        // Create resource usage for the org.
+        //
+        // NOTE: if you do this before the org is created, it'll exist as
```

Comment:
I don't follow this comment.

It seems like this should happen in a transaction, no?

Reply:
The comment was just a note (admittedly not the most clear) to myself that basically, yeah, a CTE/Transaction/Saga would be the way to go. The ordering comment was just a hint for "if I went down the saga route" that I could technically create this `resource_usage` object before the `organization` (since no one would be referencing it) rather than the other way around (and risk exposing an invalid `organization` object).

I can definitely put this in a transaction for now, however - done in 83d8d9fc97ff8c8991e0e7f1fde898efe5e54506